examples/ShoppingAds/AddShoppingSmartAd.cs

```diff
@@ -0,0 +1,398 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V2.Errors;
+using Google.Ads.GoogleAds.V2.Common;
+using Google.Ads.GoogleAds.V2.Resources;
+using Google.Ads.GoogleAds.V2.Services;
+
+using System;
+using static Google.Ads.GoogleAds.V2.Enums.AdGroupAdStatusEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.AdGroupCriterionStatusEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.AdGroupStatusEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.AdGroupTypeEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.AdvertisingChannelTypeEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.BudgetDeliveryMethodEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.CampaignStatusEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.ListingGroupTypeEnum.Types;
+using static Google.Ads.GoogleAds.V2.Resources.Campaign.Types;
+using static Google.Ads.GoogleAds.V2.Enums.AdvertisingChannelSubTypeEnum.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V2
+{
+    /// <summary>
+    /// This code example creates a standard shopping campaign, a shopping product ad group and a
```

Comment:
The comment here needs to match the description below. I believe the goal is Smart Shopping, and this may be from a standard shopping example.

Reply:
Done.